docs/reference/packages.rst

```diff
@@ -139,6 +152,23 @@ be installed automatically from the :term:`micropython-lib`. The third
 dependency installs the content as defined by the ``package.json`` file of the
 ``main`` branch of the GitHub repo ``org/micropython-additions``.
 
+The URLs in the ``package.json`` can also be specified relative to the
+directory containing the ``package.json`` file, e.g. the following
+``github:org/micropython-mlx90640/mlx90640/package.json`` can be used to
+install the ``mlx90640`` package::
+
+    {
+      "urls": [
+        ["mlx90640/__init__.py", "__init__.py"],
+        ["mlx90640/utils.py", "utils.py"],
+      ],
+      ...
+
+The use of relative URLs makes it easier to install packages from mirrors,
```

Comment:
IMO the relative URL example should replace the absolute path example, because IMO relative URLs should become the default way to do things.  Then the absolute path example should come second, as an alternative if users need to specify absolute paths.

(I feel like specifying an absolute URL is only needed in very special cases...?)

Reply:
OK. Shall do. It might be best to tweak the example a little to clarify a few things and perhaps use a real example so users can see the correct relative file layout in a github repo.

Reply:
I have updated the example in the packages docs to encourage relative URLs as default usage.